src/TestProblem/Hydro/CCSN/Record_CCSN.cpp

```diff
@@ -303,3 +346,103 @@ void Record_CCSN_GWSignal()
 
 } // FUNCTION : Record_CCSN_GWSignal()
 #endif // ifdef GRAVITY
+
+
+
+//-------------------------------------------------------------------------------------------------------
+// Function    :  Detect_CoreBounce
+// Description :  Check whether the core bounce occurs
+//
+// Note        :  1. Invoked by Record_CCSN()
+//             :  2. Based on two criteria:
+//                   --> (a) The central density is larger than 2e14
+//                       (b) Any cells within 30km has entropy larger than 3
+//-------------------------------------------------------------------------------------------------------
+void Detect_CoreBounce()
+{
+
```

Comment:
I remember that if the time step is too big in GAMER, the bounce time might be delayed a little bit. Is there any routine to automatic reduce the time step when it is closing to bounce?

Reply:
I do not know if GAMER has this feature. @hyschive 

However, I think it could be difficult to know the bounce time in advance. One solution could be that we update `DT__MAX` manually when the central density is larger than the given threshold, probably a routine invoked in `Mis_GetTimeStep_CCSN()`. I think @Inhyeok-Song can implement that in his `ccsn-delep` branch.